tools/depends/target/pythonmodule-pycryptodome/Makefile

```diff
@@ -36,6 +36,9 @@ $(PLATFORM): $(DEPS) | $(TARBALLS_LOCATION)/$(ARCHIVE).$(HASH_TYPE)
 ifeq ($(OS),android)
 	cd $(PLATFORM); patch -p1 -i ../03-android-dlopen.patch
 endif
+ifeq ($(PLATFORM),webos)
```

Comment:
can you just set this for all platforms. Testing locally looks good for apple things, and i dont believe it will have any issues on android

Reply:
And it also looks good to set it for pycryptodome as you mentioned, might as well just get them both done

Reply:
Sure, though wouldn't it need to be `.dylib` for apple?

Reply:
So is fine, and is what is currently used anyway on apple platforms for these modules